RNTester/js/OutOfTreeTestPlatform/Libraries/Components/ProgressViewIOS/ProgressViewIOS.dummy.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+'use strict';
+
+const React = require('React');
+const StyleSheet = require('StyleSheet');
+const Text = require('Text');
+const View = require('View');
+
+class DummyProgressViewIOS extends React.Component {
+  render() {
+    return (
```

Comment:
why not just have the render throw("Not implemented") or something similar? or just export `require('UnimplementedView');` like the other files?